### DISCUSSION TITLE
copy folder firmware

### DISCUSSION THREAD

**Discussion by gamoemx** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2298] [Q&A]
I am new to this and inexperienced, I want to copy a folder to update my touchscreen, can you tell me how to copy a complete folder

**Comment by Mactastic1-5** [comment]
https://user-images.githubusercontent.com/54359396/100600564-b9caed80-3301-11eb-8997-d376f05323f6.jpg

SD Card has to be formatted to FAT32 (FAT) and have a partition scheme of Master Boot Record.
